apps/condo/domains/common/schema/fields.js

```diff
@@ -131,6 +131,18 @@ const MONEY_AMOUNT_FIELD = {
     },
 }
 
+const POSITIVE_MONEY_AMOUNT_FIELD = {
+    ...MONEY_AMOUNT_FIELD,
+    type: SignedDecimal,
+    template: 'positive',
+}
+
+const NON_NEGATIVE_MONEY_FIELD = {
+    ...MONEY_AMOUNT_FIELD,
+    type: SignedDecimal,
+    template: 'non-negative',
+}
+
```

Comment:
Why not unite them? Dont know the motivation

Reply:
I think it would also be cool if we specify which one of `POSITIVE MONEY AMOUNT FIELD` or `NON NEGATIVE ...` has `zero`

Reply:
It would also be cool if you tell about this on tech-talk

Reply:
You can use `signedDecimal type + template` for cases: `>0` (positive), `>=0` (non-negative), `<0` (negative), `<=0` (non-positive). This ones was created just as "most used template"